app/javascript/mastodon/components/skeleton.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+interface Props {
+  width: number | string;
+  height?: number | string;
```

Comment:
Why is one possibly null but not the other?

Reply:
This is a mistake and has been corrected.